src/lib/config.js

```diff
@@ -201,8 +200,6 @@ function getLocalConfig () {
     Number(Config.getEnv(envPrefix, 'ROUTE_CLEANUP_INTERVAL')) || DEFAULT_ROUTE_CLEANUP_INTERVAL
   const routeExpiry =
     Number(Config.getEnv(envPrefix, 'ROUTE_EXPIRY')) || DEFAULT_ROUTE_EXPIRY
-  const routeShift =
```

Comment:
The changes should make the `routeShift` option unnecessary, so removing it doesn't actually break anything.